src/astra_driver.cpp

```diff
@@ -181,13 +199,16 @@ void AstraDriver::getAvailabilityDiagnostic(diagnostic_updater::DiagnosticStatus
 
   if (device_count == 0)
   {
-    stat.summary(DiagnosticStatus::ERROR, "No Device connected");
+    diagnostics_updater_.broadcast(DiagnosticStatus::ERROR, "No Device connected");
```

Comment:
This is interfering with the rest of the diagnostics. It's the responsibility of the availability diagnostic to publish "No device connected". The others will eventually go in error if there is no device, but even if that wasn't the case, that's ok. At a next stage comes the aggregator, and if one of the diagnostics is in error, the aggregator will put the aggregated camera diagnostic in error (this is application level configuration).

Reply:
That's true, and I thought here that if a device is not connected, we are not interested about frequency anymore - "No device connected" is enough to get the first idea about what is wrong. Of course, we can still leave the frequency diagnostics as is (it will most probably output "No events recorded", which is okay together with "No device connected", if someone will dig deeper into the aggregated diagnostics).

Reply:
So let's revert this